portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,4 +38,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Gson gson = new Gson();
     response.getWriter().println(gson.toJson(store.getComments()));
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String userName = getParameter(request, "name", "Anonymous");
+    String comment =  getParameter(request, "comment", "");
+
+    store.post(new Comment(userName, comment));
```

Comment:
Is return value supposed to be unused?

Reply:
I anticipated that the datastore may return a boolean value on a successful insert, but that doesn't seem to be the case. In task 5, I have rectified this issue.